configure.ac

```diff
@@ -121,6 +133,35 @@ AS_IF([test "$enable_threads" != "no"], [
        ])
 AM_CONDITIONAL([HAVE_PTHREADS], [test "$HAVE_POSIX_THREADS" = "1"])
 
+AS_IF([test "$with_hwloc" != "no"], [
```

Comment:
Does it also work with a custom path given to `--with-hwloc`?

Reply:
It does. I've checked on Borealis that passing a custom path (in this case `/home/philipma/multirail/hwloc/install`) to `--with-hwloc` works. Along with the expected text being output during the configure phase indicating that hwloc is being used from the provided path, a call to `ldd ./test/unit/hello` shows that **libhwloc** is being sourced from the provided path

![image](https://github.com/Sandia-OpenSHMEM/SOS/assets/97559536/775a655e-da1a-4f16-99aa-46c8f76df72f)